support-frontend/assets/components/nav/nav.tsx

```diff
@@ -15,12 +15,14 @@ interface NavProps {
 	countryGroupIds: CountryGroupId[];
 	selectedCountryGroup: CountryGroupId;
 	subPath: string;
+	countryIsAffectedByVATStatus: boolean;
 }
 
 function Nav({
 	countryGroupIds,
 	selectedCountryGroup,
 	subPath,
+	countryIsAffectedByVATStatus,
```

Comment:
And this as 

```suggestion
	countryIsAffectedByVATStatus = false,
```

To avoid having to pass it through

Reply:
@jamesgorrie suggestions looks good, thanks!